WcaOnRails/.rubocop.yml

```diff
@@ -54,3 +54,9 @@ Style/SignalException:
 Style/ClassVars:
   Exclude:
     - app/models/country.rb
+
+Style/IfInsideElse:
+  Enabled: false
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
```

Comment:
Good call! Thank you @vincentwoo and @jonatanklosko I couldn't resist adding a `unskipped` method, so the solution becomes:

```ruby
solve_times.drop_while(&:unskipped?).all?(&:skipped?)
```